### ISSUE TITLE
Seemingly incorrect const size in fs base calculation (x86-64)

### ISSUE THREAD

**Issue by r4dr3fr4d** [angr/pypcode#32]
```
~ % ./test_pcode.py -r x86:LE:64:default ~/test.bin
--------------------------------------------------------------------------------

00000000/9: 64 48 8b 14 25 c0 ff ff ff MOV RDX,qword ptr FS:[0xffffffc0]
--------------------------------------------------------------------------------
  0: unique[0x4f00:8] = INT_ADD register[0x110:8], const[0xffffffc0:8]
  1: unique[0xc000:8] = LOAD const[0x55e5781c94a0:8], unique[0x4f00:8]
  2: register[0x10:8] = COPY unique[0xc000:8]
```

test.bin contains just those 9 bytes of that single MOV instruction.  Notice in the first pcode op, the const[0xffffffc0:8] is 8 bytes long, but shouldn't it be only 4?  Or extended first?

Not sure if this is a sleigh bug?

**Comment by r4dr3fr4d** [comment]
https://github.com/NationalSecurityAgency/ghidra/issues/4310

Looks to be, does this repo need to be updated or am I missing something?

**Comment by mborgerson** [comment]
@r4dr3fr4d: Apparently the fix for the issue you mentioned is merged into master branch and will be in 10.2 release, which I expect will be coming very soon. pypcode is on 10.1.4 currently. When 10.2 drops I'll merge in the updates.

**Comment by mborgerson** [comment]
10.2.2 is merged, so this should be fixed now. Testing with your provided binary shows proper sign-extension:

```
--------------------------------------------------------------------------------
00000000/9: MOV RDX,qword ptr FS:[-0x40]
--------------------------------------------------------------------------------
  0: unique[0x4f00:8] = INT_ADD register[0x110:8], const[0xffffffffffffffc0:8]
  1: unique[0xc000:8] = LOAD const[0x2857df0:8], unique[0x4f00:8]
  2: register[0x10:8] = COPY unique[0xc000:8]
```

Thanks for reporting
